doc/contributing/submitting-changes.chapter.md

```diff
@@ -234,6 +234,15 @@ digraph {
 
 The `master` branch is the main development branch. It should only see non-breaking commits that do not cause mass rebuilds.
 
+The guideline for proposed change is good for `master` branch:
+
+- Rebuild count is less than 1000 dependent 1000.
+- Rebuild count is between 1000 and 2000 lightweight packages (like python depends) and does not contain heavyweight package rebuilds (like `chromium` or `firefox`).
+- Kernel updates (they usually trigger about 2000 rebuilds).
+- Security fixes.
```

Comment:
Also really depends on the severity. If the exploitation is very unlikely and not practical at all we don't want to rebuild the entire world. For example there are sometimes CVEs which need root to be exploitet which in my opinion do not justify rebuilds over 5000.

Reply:
Changed to somewhat vague `Urgent security fixes. `. People doing security fixes probably know the implications and finer details.